widgets/cdatetime/org.eclipse.nebula.widgets.cdatetime/src/org/eclipse/nebula/widgets/cdatetime/CDateTime.java

```diff
@@ -1527,9 +1524,7 @@ protected void postClose(Shell popup) {
 	 */
 	public void removeSelectionListener(SelectionListener listener) {
 		if (listener != null) {
-			TypedListener l = new TypedListener(listener);
-			removeListener(SWT.Selection, l);
-			removeListener(SWT.DefaultSelection, l);
+			addTypedListener(listener, SWT.Selection, SWT.DefaultSelection);

```

Comment:
This definitely looks wrong.

Reply:
Oops. Looks like a C&P error.
Will fix that, thanks for spotting!